### PR TITLE
build 0.0.9

### PR DESCRIPTION
**Pull request by boldorider4**
* [x] upstream [requirements](https://github.com/pymc-devs/pymc-experimental/blob/main/requirements.txt)
* [x] checked [license](https://github.com/pymc-devs/pymc-experimental/blob/main/LICENSE)
* [x] created package from scratch with `conda-skeleton pypi pymc-experimental`

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+  # skip because arviz is not supported for py38
+  skip: True  # [py<39]
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+
+requirements:
+  host:
+    - pip
+    - pymc 5.6.1
+    - python
+    - scikit-learn
```
Comment (cbouss) [inline recipe/meta.yaml]:
```suggestion
```
This doesn't look to be required at build time.

File: recipe/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+build:
+  number: 0
+  # skip because arviz is not supported for py38
+  skip: True  # [py<39]
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+
+requirements:
+  host:
+    - pip
+    - pymc 5.6.1
```
Comment (cbouss) [inline recipe/meta.yaml]:
```suggestion
```

This doesn't look to be required at build time.

File: recipe/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+    - pymc_experimental.tests.model_transform
+    - pymc_experimental.tests.utils
+    - pymc_experimental.utils
+  requires:
+    - pip
+  commands:
+    - pip check
+
+about:
+  home: https://www.pymc.io/projects/experimental/en/latest
```
Comment (cbouss) [inline recipe/meta.yaml]:
```suggestion
  home: https://www.pymc.io/projects/experimental
```

**Review by ryanskeith — COMMENTED** [review]
I agree with @cbouss  comments.

File: README.md [hunk 4]
```diff
@@ -1 +1,8 @@
 # About pymc-experimental-feedstock
+
+[Home](https://www.pymc.io/projects/experimental/en/latest/)
+Feedstock license: BSD 3
```
Comment (skupr-anaconda) [inline README.md]: You need to add the feedstock license file like here https://github.com/AnacondaRecipes/medspacy_quickumls-feedstock/blob/main/LICENSE

File: recipe/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,67 @@
+{% set name = "pymc-experimental" %}
+{% set version = "0.0.9" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
```
Comment (skupr-anaconda) [inline recipe/meta.yaml]:
```suggestion
  name: {{ name|lower }}
  version: {{ version }}
```

File: recipe/meta.yaml [hunk 6]
```diff
@@ -0,0 +1,67 @@
...
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
```
Comment (skupr-anaconda) [inline recipe/meta.yaml]:
```suggestion
  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
```

File: recipe/meta.yaml [hunk 7]
```diff
@@ -0,0 +1,67 @@
...
+    - pip
+  commands:
+    - pip check
+
+about:
+  home: https://www.pymc.io/projects/experimental/en/latest
+  license: Apache-2.0
+  license_family: APACHE
+  license_file: LICENSE
+  summary: the next batch of cool PyMC features
```
Comment (skupr-anaconda) [inline recipe/meta.yaml]:
```suggestion
  summary: The next batch of cool PyMC features
```

File: recipe/meta.yaml [hunk 8]
```diff
@@ -0,0 +1,67 @@
...
+  description: |
+    A home for new additions to PyMC, which may include unusual probability distribitions, advanced model fitting
+    algorithms, or any code that may be inappropriate to include in the pymc repository, but may want to be made
+    available to users.
+  doc_url: https://github.com/pymc-devs/pymc-experimental
```
Comment (skupr-anaconda) [inline recipe/meta.yaml:58]:
```suggestion
  doc_url: https://www.pymc.io/projects/experimental
```

File: recipe/meta.yaml [hunk 9]
```diff
@@ -0,0 +1,67 @@
...
+  dev_url: https://github.com/pymc-devs/pymc-experimental
+
+extra:
+  recipe-maintainers:
+    - boldorider4
+  skip-lints:
+    - invalid_url
+    - host_section_needs_exact_pinnings
```
Comment (skupr-anaconda) [inline recipe/meta.yaml]:
```suggestion
```
If we remove `pymc 5.6.1` from host then this skip is redundant

File: recipe/meta.yaml (hunk 8 above)
Comment (boldorider4) [reply recipe/meta.yaml:58]: to me the GitHub project looks a more comprehensive in terms of documentation than the project main page.

Comment (skupr-anaconda) [reply recipe/meta.yaml:58]: Ah, you are right. Closing this comment

**Review by skupr-anaconda — APPROVED** [review]
LGTM

**Review by cbouss — APPROVED** [review]
Small comments, before merging can you add setuptools to host and remove abs.yaml?

Otherwise, LGTM

**Pull request merged**